packages/jest/test/react-enzyme.test.js

```diff
@@ -143,6 +143,19 @@ const cases = {
       )
     }
   },
+  'using the css helper': {
```

Comment:
OTOH this one holds an array so I would assume that maybe the added test titles should be swapped but I'm also not sure what "using the css helper" test would be about - could you make this test title a little bit more descriptive?

Reply:
thanks, let me know if the new description is missing anything